features/api_security.feature

```diff
@@ -29,7 +29,16 @@ Feature: API security/security status tests
         """
         When I run `apt update` with sudo
         And I run `apt upgrade -y` with sudo
-        And I run `apt install jq bzip2 libopenscap8 -y` with sudo
+        And I run `apt install jq bzip2 -y` with sudo
+        # Install the oscap version 1.3.7 which solved the epoch error message issue
+        And I run `apt-get install lcov swig xsltproc rpm-common lua5.3 libyaml-dev libapt-pkg-dev libdbus-1-dev libdbus-glib-1-dev libcurl4-openssl-dev libgcrypt-dev libselinux1-dev libgconf2-dev libacl1-dev libblkid-dev libcap-dev libxml2-dev libxslt1-dev libxml-parser-perl libxml-xpath-perl libperl-dev librpm-dev librtmp-dev libxmlsec1-dev libxmlsec1-openssl cmake make build-essential -y` with sudo
+        And I run `apt-get remove *rpm* -y` with sudo
```

Comment:
Rather than removing packages immediately after installing them, we can probably just avoid installing them - hopefully my above comment will help

Reply:
Yes, I was still thinking about how to go about this. Thanks for the doc link.